python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -386,6 +443,55 @@ def predict(self, X):
            Dense vector (int) of shape (n_samples, 1)
 
         """
+        for w in self.workers:
+            if self.rfs[w].result().multi_class == 1:
```

Comment:
I think these python changes are not supposed to be part of this PR? The PR is just supposed to be the C++ side, right?